### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by paulinagonzalezc**
For this milestone of my portfolio website, I implemented the mobile menu using my recently acquired JavaScript knowledge.

* I added the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
* When the user clicks (or taps) the hamburger button, the mobile menu appears.
* When the user clicks (or taps) the close (X) button, the mobile menu disappears.
* When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
* When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
* Stick to the design as much as possible.
* NOTE: My coding partner left during the day and they just assigned me to another team, that’s why there is no contribution.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (Munsa1) [inline .eslintrc.json:1]: - [ ] Kindly fix your linters, I dentified the problem as to why the `Eslint` linters are not showingin the checks
image
- You need to update you `linters.yml` file

Your file
image

Original file
image
.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const mobileBtn = document.getElementById("mobile-cta");
+nav = document.querySelector("nav");
+mobileBtnExit = document.getElementById("mobile-exit");
+portfolioBtn = document.getElementById("menu-portfolio");
+aboutBtn = document.getElementById("menu-about");
+contactBtn = document.getElementById("menu-contact");
+
+mobileBtn.addEventListener("click", () => {
```
Comment (Munsa1) [inline index.js:8]: - [ ] Kindly fix the overflow on your design so that the whole mobile menu can be visible

Your design
image

Given design
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @paulinagonzalezc, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 2 above)
Comment (paulinagonzalezc) [reply index.js]: @Munsa1 Hello Munsa, are you sure it loads like that? This is how it loads on my browser:
Screen Shot 2022-10-10 at 19 32 33

**Review by franclobo — APPROVED** [review]
Hi @paulinagonzalezc, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️
- Correct Github flow ✔️
- Descriptive PR ✔️
- Professional README file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**